src/ms-review.tex

```diff
@@ -1775,12 +1774,12 @@ \subsection{Software Distribution}
 Zenodo\footnote{\url{https://doi.org/10.5281/zenodo.4701488}} digital libraries for
 software. The Zenodo record is synchronised with the codebase \github repository
 so that every release triggers the update of the versioned record. In addition,
-the next release of \gammapy will be added to the Open-source scientific
+ \gammapy has been added to the Open-source scientific
```

Comment:
- related comment :
Another item that could be mentioned in the applications section is that GammaPy has been added to the OSSR catalogue of the ESCAPE and EOSC. There is a paper currently written that could be linked from here.

Should we add a reference to https://arxiv.org/pdf/2212.00499.pdf or to the paper in prep ? @vuillaut

Reply:
The paper in prep would be a better reference. The ref should be available very soon if that can wait a week or so :)
Thanks

Reply:
We plan to submit during the co-working week starting March 20th, let's see who is faster... :-)

Reply:
Hi
The OSSR paper is now published.
Could you add the following reference please?
```
Vuillaume T, Al-Turany M, Füßling M et al. The ESCAPE Open-source Software and Service Repository [version 1; peer review: awaiting peer review]. Open Res Europe 2023, 3:46 (https://doi.org/10.12688/openreseurope.15692.1)
```

Reply:
added in #129

Reply:
> let's see who is faster... :-)

You won...😂